src/Main.java

```diff
@@ -0,0 +1,90 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        int[] list = {2, 3, 1, 0};
+        System.out.println(sumNumbers(list));
+
+        List<Integer> listTwo = new ArrayList<>();
+        listTwo.add(2);
+        listTwo.add(3);
+        listTwo.add(1);
+        listTwo.add(0);
+
+        System.out.println(avarageNumbers(listTwo));
+
+        multiplicationTable(15);
+
+        numbersCount();
+
+        List<String> letterList = new ArrayList<>();
+        letterList.add("a");
+        letterList.add("b");
+        letterList.add("c");
+
+        List<String> numberList = new ArrayList<>();
+        numberList.add("1");
+        numberList.add("2");
+        numberList.add("3");
+
+        System.out.println(combineList(letterList, numberList));
+
+    }
+
+    public static int sumNumbers(int[] list) {
+        int total = 0;
+        for (int i = 0; i < list.length; i++) {
+            total += list[i];
+        }
+        return total;
+    }
+
+    public static double avarageNumbers(List<Integer> list) {
+        int total = 0;
+        for (int i = 0; i < list.size(); i++) {
+            total += list.get(i);
+        }
+        return (double) total / list.size();
+
+
+    }
+
+    public static void multiplicationTable(int n) {
+        int count = 1;
+        while (count <= 15) {
+            System.out.println(n * count);
+            count++;
+        }
```

Comment:
This isn't exactly correct
If n is 15
You need to print to console
15 * 1 = 15
15 * 2 = 30
etc

Reply:
done